rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -853,25 +853,19 @@ namespace rsx
 			u32 out_pitch = 0;
 			u32 out_alignment = 64;
 
-			switch (method_registers.blit_engine_context_surface())
+			if (method_registers.blit_engine_context_surface() == blit_engine::context_surface::surface2d)
 			{
-			case blit_engine::context_surface::surface2d:
 				dst_dma = method_registers.blit_engine_output_location_nv3062();
 				dst_offset = method_registers.blit_engine_output_offset_nv3062();
 				dst_color_format = method_registers.blit_engine_nv3062_color_format();
 				out_pitch = method_registers.blit_engine_output_pitch_nv3062();
 				out_alignment = method_registers.blit_engine_output_alignment_nv3062();
-				break;
-
-			case blit_engine::context_surface::swizzle2d:
+			}
+			else
```

Comment:
not sure why the switch was removed, allows invalid context_surface to get through here instead of throwing or showing error like before. The context value is 32 bits and has only 2 usable values, unless you confirmed that invalid context defaults to swizzle shuffle.

Reply:
its already handled in the exception handler when converting to blit_engine::context_surface, so adding a default case here is no-op.

Reply:
those to_xxxxx methods will be removed down the line, so all code going forward should assume they have no checks.